versioneer.py

```diff
@@ -1,4 +1,5 @@
 # Version: 0.20
+# versioneer is on 0.29, would be good to update
```

Comment:
These manual updates are the worst part of vendoring things! Would this comment be a good thing to turn into an issue?